tests/test_plugins/test_plugins.py

```diff
@@ -64,14 +68,18 @@ def cfg_file_create(cfg_file_contents, path):
     return file_loc
 
 
-def test_plugin_warning(tmp_path, python_version):
-    cfg_file_contents = textwrap.dedent(
-        """
-        [CLI]
-        plugins = DNEPlugin
-    """
+@pytest.fixture
+def random_string():
+    all_letters = string.ascii_lowercase
+    a = random.Random()
```

Comment:
**Comment** : I could be cool to use a seed in a constant here (that could be by default None) so one can reproduce a test suite.

Reply:
having it as a parameter may be difficult with how fixtures work